containers/server-image/Dockerfile

```diff
@@ -9,126 +9,77 @@ ARG PRODUCT_PATTERN_PREFIX="patterns-uyuni"
-COPY pam-susemanager /etc/pam.d/susemanager
-COPY pam-susemanager-account /etc/pam.d/susemanager-account
-COPY pam-susemanager-auth /etc/pam.d/susemanager-auth
-COPY pam-susemanager-password /etc/pam.d/susemanager-password
-COPY pam-susemanager-session /etc/pam.d/susemanager-session
-COPY pam-password-auth /etc/pam.d/password-auth
-COPY pam-postlogin /etc/pam.d/postlogin
-COPY pam-system-auth /etc/pam.d/system-auth
-COPY nsswitch.conf /etc/nsswitch.conf
-
-RUN chmod -R 755 /usr/bin/timezone_alignment.sh
-
-# Set a warning message for interactive shells
-COPY motd /etc/motd
-RUN echo "cat /etc/motd" >/etc/sh.shrc.local
-
-RUN systemctl enable prometheus-node_exporter; \
-    systemctl enable uyuni-setup; \
-    systemctl enable timezone_alignment; \
-    systemctl enable sssd;
-
-# Provide tool to synchronize package and configuration files to persistent volumes
-COPY uyuni-configfiles-sync /usr/bin
-RUN chmod -R 755 /usr/bin/uyuni-configfiles-sync
+RUN echo "rpm.install.excludedocs = yes" >>/etc/zypp/zypp.conf && \
+    zypper ref && zypper --non-interactive up && \
+    zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-licenses --force-resolution \
+        ${EXTRAPACKAGES} \
+        ${PRODUCT_PATTERN_PREFIX}_server \
+        ${PRODUCT_PATTERN_PREFIX}_retail \
+        billing-data-service \
+        grub2-x86_64-efi \
+        grub2-arm64-efi \
+        grub2-powerpc-ieee1275 \
+        ed \
+        susemanager-tftpsync \
+        golang-github-prometheus-node_exporter \
+        prometheus-postgres_exporter \
+        golang-github-QubitProducts-exporter_exporter \
+        prometheus-jmx_exporter \
+        prometheus-jmx_exporter-tomcat \
+        spacecmd \
+        grafana-formula \
+        locale-formula \
+        prometheus-exporters-formula \
+        prometheus-formula \
+        registry-formula \
+        virtualization-formulas \
+        uyuni-config-formula \
+        saphanabootstrap-formula \
+        sapnwbootstrap-formula \
+        drbd-formula \
+        habootstrap-formula \
+        saline-formula \
+        salt-shaptools \
+        inter-server-sync \
+        golang-github-prometheus-node_exporter \
+        prometheus-postgres_exporter \
+        golang-github-QubitProducts-exporter_exporter \
+        prometheus-jmx_exporter \
+        spacecmd \
+        javamail \
+        virtual-host-gatherer-libcloud \
+        virtual-host-gatherer-Libvirt \
+        virtual-host-gatherer-Nutanix \
+        virtual-host-gatherer-VMware \
+        vim \
+        python3-pygit2 \
+        ipmitool \
+        sssd \
+        sssd-dbus \
+        sssd-ldap \
+        sssd-ad \
+        sssd-ipa \
+        sssd-krb5 \
+        sssd-tools && \
+    sed -i 's/sysctl kernel.shmmax/#sysctl kernel.shmmax/g' /usr/bin/uyuni-setup-reportdb && \
+    systemctl enable prometheus-node_exporter && \
+    systemctl enable sssd
+
+ADD root.tar.gz /
```

Comment:
With these changes means that we cannot just call podman build directly with this dockerfile, but must use the release script to create the tar.gz file first?

Reply:
Yes! But it seems no one ever built the containers directly from their local machine... So why getting the pain of additional layers when we can get rid of them?

Reply:
Can we run just `ADD root /´ ? I made a quick simple test locally, and it looks the files where copied. Do we need some extra transformation to create the compressed file?